docs/pages/guides/testing-rsc.mdx

```diff
@@ -64,36 +66,32 @@ it(`renders to RSC`, async () => {
 });
 ```
 
-Any code you import in your tests will be run in the server environment, so you can import server-only modules like `react-server` and `server-only`. This is useful for determining if a library is compatible with React Server Components.
+Any code you import in your test files will run in the server environment. You can import server-only modules like `react-server` and `server-only`. This is useful for determining if a library is compatible with RSC.
 
 ### Custom expect matchers
 
-Jest Expo for RSC adds a couple of custom matchers to Jest's expect:
+`jest-expo` for RSC adds a couple of custom matchers to Jest's `expect`:
 
 - `toMatchFlight`: Render a JSX element using a pseudo-implementation of the render in Expo CLI and compare to a flight string.
 - `toMatchFlightSnapshot`: Same as `toMatchFlight` but saves the flight string to a snapshot file.
 
-Behind the scenes, these methods are performing part of the framework operation required to render React Server Components. The component is streamed to a string and compared all at once. You can alternatively stream it manually to observe the progress.
+Behind the scenes, these methods handle a part of the framework operation needed to render RSC. The component is streamed to a string and compared all at once. You can alternatively stream it manually to observe the progress.
```

Comment:
I think the key word here is "buffer" rather than "stream". The point is that something collects the whole output, converting it from a stream to a complete datum.

```suggestion
Behind the scenes, these methods handle a part of the framework operation needed to render RSC. The component's render stream is buffered to a string and compared all at once. You can alternatively stream it manually to observe the rendering progress.
```